### PR TITLE
feat: add bicep support for ai

### PR DESCRIPTION
**Pull request by frankliu20**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: infra/bicep/modules/ai/openai.bicep [hunk 1]
```diff
@@ -0,0 +1,65 @@
+param accountName string
+param location string
```
Comment (sonwan2020) [inline infra/bicep/modules/ai/openai.bicep:2]: please add some descriptions for params

File: infra/bicep/modules/ai/openai.bicep [hunk 2]
```diff
@@ -0,0 +1,65 @@
+param accountName string
+param location string
+param modelTextEmbeddingAda002 string = 'text-embedding-ada-002'
+param modelGpt4 string = 'gpt-4'
+param modelFormat string = 'OpenAI'
+param appPrincipalId string
+param roleDefinitionId string = '5e0bd9bd-7b93-4f28-af87-19fc36ad61bd' // Cognitive Services OpenAI User role
```
Comment (sonwan2020) [inline infra/bicep/modules/ai/openai.bicep:7]: add doc link to this role definition

File: infra/bicep/modules/ai/openai.bicep (hunk 1 above)
Comment (frankliu20) [reply infra/bicep/modules/ai/openai.bicep:5]: Added

File: infra/bicep/modules/ai/openai.bicep (hunk 2 above)
Comment (frankliu20) [reply infra/bicep/modules/ai/openai.bicep]: Added

**Review by sonwan2020 — APPROVED** [review]

**Pull request merged**